.gitignore

```diff
@@ -12,6 +12,7 @@ generated/
 -trace.log
 *.lst
 .dub
+dub_describe.json
```

Comment:
What's that ? It's not output by `dub` and I couldn't find a reference in this repository.

Reply:
I don't know. I'll remove it from the PR though since it's unrelated. I do know that it keeps popping up and has started doing so lately. For me anyway.

Reply:
Found out where - it's from my flycheck-dmd-dub Emacs package which got a PR merged recently.

Reply:
You might want https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/